glotaran/model/attribute.py

```diff
@@ -155,7 +151,7 @@ def decorator(cls):
 def _create_add_type_func(cls):
     @classmethod
     @wrap_func_as_method(cls)
-    def add_type(cls, type_name: str, type: Type):
+    def add_type(cls, type_name: str, type: type):
```

Comment:
I think it remains problematic, to use what is practically a built-in used also for typing as a variable.
Something to address in a next refactor @joernweissenborn

Reply:
this is the type of the item, it will be added to the __annotations__, I don't see any problem as long `type` is the same meaning as `typing.Type`